src/discrete-layer/components/layer-details/validate.ts

```diff
@@ -45,7 +45,7 @@ enforce.extend({
   },
 });
 
-const suite = (fieldDescriptor: FieldConfigModelType[], data: Record<string, unknown> = {}): any => {
+const suite = (fieldDescriptor: FieldConfigModelType[], data: Record<string, unknown> = {}): unknown => {
```

Comment:
not sure that we need to return something
line #139 ```return validate
it return function

Reply:
No it cannot be void. 
It doesn't return a function but an object returned from vest.create (lint 85 - validate.ts)